docs/content/migration.md

```diff
@@ -0,0 +1,535 @@
+`flex`, `show`, `hide`, etc.
+
+#### Angular Layout
+
+This comprehensive layout system was moved into its own project named [Angular Layout][120] 
+(`@angular/flex-layout`). 
+
+The package contains the following:
+- A [Declarative API][116] that includes attributes like `fxLayout`, `fxFlex`, `fxHide`, `fxShow`,
+  and more.
+- A [Responsive API][117] that includes attributes like `fxLayout.lt-md`, `fxFlex.gt-sm`,
+  `fxHide.gt-xs`, `[ngClass.sm]`, `[ngStyle.xs]`, and more.
+- Both APIs include [CSS Grid][65] features in addition to the CSS Flexbox features from
+  AngularJS Material.
+- The ability to define [Custom Breakpoints][118]
+- A [MediaObserver Service][119] that allows subscribing to `MediaQuery` activation changes
+
+The [Angular Layout][120] library's full UMD bundle is around `275` KBs. This size can be reduced by
+consuming the various modules and using the Angular CLI to remove unused library code, but this
+may be more complexity than you need.
+
+#### CDK Layout
+
+The Angular CDK's [Layout][121] module provides:
+- A set of `Breakpoints` that match the [Material Design][122] responsive layout grid breakpoints
+- A `BreakpointObserver` Service that provides
+    - the ability to check media query matching state via `isMatched()`
+    - the ability to subscribe to a stream of events based on changes to an array of media queries
+      that are passed into `observe()`
+- A `MediaMatcher` Service that provides access to the low level, native [`MediaQueryList`][123]
+
+If you are building responsive components or need responsive features in your application with a
+minimal increase in bundle size, then the CDK's Layout module is an option you should consider.
+
+### Typography
+
+AngularJS Material [typography classes][107] are generally identical to the Angular Material
+[typography classes][27].
+
+Angular Material also adds the following typography features:
+- [Customization via Sass mixins][174] that are similar to the theme configuration mixins
+- Utility Sass mixins and functions for use in [styling your custom components][175]
+  
+## Comparison of Features and Migration Tips
+
+Angular Material introduces a number of [new components](#new-components) and features that do not
+exist in AngularJS Material. Angular Material also makes greater use of semantic HTML and native
+elements to improve accessibility.
+
+However, some directives and services have not been implemented in Angular Material. These are
+indicated by an `N/A` in the New Docs column of the feature tables found below. 
+
+The Notes column, in each row of the tables, includes information like the new name of the feature
+or an alternative approach to implementing the feature. If the notes are blank, then the name of
+the old feature and the new feature are nearly identical (the APIs are likely to be different).
+
+### Directives
+Most directives changed to components. These make for straight forward conversions, for instance
```

Comment:
The fact that everything in AngularJS implements its components as "directives" is just an implementation detail. I would put this more like
```markdown
Most components use the same name between the two libraries, changing the `md-` prefix to `mat-`. For example, `<md-card>` becomes `<mat-card>`.
```